doc/source/reference/arrays.classes.rst

```diff
@@ -43,6 +43,10 @@ NumPy provides several hooks that classes can customize:
 
    .. versionadded:: 1.13
 
+   .. note:: The API is `provisional
+             <https://docs.python.org/3/glossary.html#term-provisional-api>`_,
```

Comment:
That's strange: I did try `make html` and it looked OK:
![spectacle i11318](https://cloud.githubusercontent.com/assets/2789820/24323072/90f58900-1146-11e7-9b38-5dc33f7ceec2.png)